### DISCUSSION TITLE
Login Failed error with correct credentials

### DISCUSSION THREAD

**Discussion by jimbarrsc** [thesamim/TickTickSync#141] [General]
I used to be able to successfully log in to TickTick through the plugin, but now, I consistently get this error on all Obsidian platforms I try to use:

![image](https://github.com/thesamim/TickTickSync/assets/127752867/48e3e2d0-7851-4e55-9e8a-437bb7ca3003)

My TickTick password changed a couple weeks ago, and I can't get the plugin to successfully log in.

I uninstalled this plugin and deleted the tickticksync folder in the .obsidian/plugins folder but when I reinstall, the plugin still gives the same error.

I can successfully log in to TickTick through a web browser using the same username and password as I use with the plugin.

Thoughts?

**Comment by HermesT2** [comment]
I am having the same problem. its shows:

```
Login Failed. 
"operation":"Get Inbox Properties"
"statusCode":503
"errorMessage":"&lt;html&gt;\r\n&lt;head&gt;&lt;title&gt;503 Service Temporarily Unavailable&lt;/title&gt;&lt;/head&gt;\r\n&lt;body&gt;\r\n&lt;center&gt;&lt;h1&gt;503 Service Temporarily Unavailable&lt;/h1&gt;&lt;/center&gt;\r\n&lt;/body&gt;\r\n&lt;/html&gt;\r\n&lt;!-- a padding to disable MSIE and Chrome friendly error page --&gt;\r\n&lt;!-- a padding to disable MSIE and Chrome friendly error page --&gt;\r\n&lt;!-- a padding to disable MSIE and Chrome friendly error page --&gt;\r\n&lt;!-- a padding to disable MSIE and Chrome friendly error page --&gt;\r\n&lt;!-- a padding to disable MSIE and Chrome friendly error page --&gt;\r\n&lt;!-- a padding to disable MSIE and Chrome friendly error page --&gt;\r\n"}
```


I think the Service Temporarily Unavailable might point to the issue.

**Comment by jbarr** [comment]
Looks like this should have been in Issues: https://github.com/thesamim/TickTickSync/issues/143

That solved it. Thank you!

**Comment by thesamim** [comment]
1.0.34 is the latest version attempting to fix this.
